### PR TITLE
Add PyOrgMode

### PR DESCRIPTION
**Pull request by jschueller**

### CODE REVIEW

File: recipes/pyorgmode/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+source:
+  fn: PyOrgMode-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/4f/81/e403d3c062000481e5d0b54167369a7b4d33a6e001ccc441bbefeb611600/PyOrgMode-0.1.tar.gz
+  md5: 51db078517762018a8be341a0be71394
+
+build:
+  number: 0
+  # cannot use --single-version-externally-managed because distutils is used
+  script: python setup.py install --record record.txt
```
Comment (jakirkham) [inline recipes/pyorgmode/meta.yaml]: If `distutils` is used, please drop `--record record.txt`.

File: recipes/pyorgmode/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+requirements:
+  build:
+    - python
+    - setuptools
```
Comment (jschueller) [inline recipes/pyorgmode/meta.yaml]: ok

File: recipes/pyorgmode/meta.yaml (hunk 1 above)
Comment (jschueller) [inline recipes/pyorgmode/meta.yaml]: ok

File: recipes/pyorgmode/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+  version: {{ version }}
+
+source:
+  fn: PyOrgMode-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/4f/81/e403d3c062000481e5d0b54167369a7b4d33a6e001ccc441bbefeb611600/PyOrgMode-0.1.tar.gz
+  md5: 51db078517762018a8be341a0be71394
+
+build:
+  number: 0
+  # cannot use --single-version-externally-managed --record record.txt because distutils is used
```
Comment (jakirkham) [inline recipes/pyorgmode/meta.yaml]: Also we can drop this comment.

File: recipes/pyorgmode/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,37 @@
+{% set version = "0.1" %}
+
+package:
+  name: pyorgmode
+  version: {{ version }}
+
+source:
+  fn: PyOrgMode-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/4f/81/e403d3c062000481e5d0b54167369a7b4d33a6e001ccc441bbefeb611600/PyOrgMode-0.1.tar.gz
```
Comment (jakirkham) [inline recipes/pyorgmode/meta.yaml]: This is very fragile for updating. Please change this to the following (this requires templating `name`).

```yaml
  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
```

xref: https://github.com/conda-forge/staged-recipes/blob/a0222417ed30935a383f5773ff21d23322247cdd/recipes/example/meta.yaml#L18

File: recipes/pyorgmode/meta.yaml (hunk 3 above)
Comment (jschueller) [inline recipes/pyorgmode/meta.yaml]: ok

File: recipes/pyorgmode/meta.yaml (hunk 4 above)
Comment (jschueller) [inline recipes/pyorgmode/meta.yaml]: ok

File: recipes/pyorgmode/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+  md5: 51db078517762018a8be341a0be71394
```
Comment (jschueller) [inline recipes/pyorgmode/meta.yaml]: ok

File: recipes/pyorgmode/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,37 @@
+{% set version = "0.1" %}
+{% set name = "pyorgmode" %}
```
Comment (jakirkham) [inline recipes/pyorgmode/meta.yaml]: Needs to be `PyOrgMode`.

File: recipes/pyorgmode/meta.yaml [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+
+package:
+  name: pyorgmode
```
Comment (jakirkham) [inline recipes/pyorgmode/meta.yaml]: Can replace with `{{ name|lower }}`.

File: recipes/pyorgmode/meta.yaml [hunk 8]
```diff
@@ -0,0 +1,37 @@
...
+  version: {{ version }}
+
+source:
+  fn: PyOrgMode-{{ version }}.tar.gz
```
Comment (jakirkham) [inline recipes/pyorgmode/meta.yaml]: Can replace with `{{ name }}-{{ version }}.tar.gz`.

Comment (jschueller) [inline recipes/pyorgmode/meta.yaml]: ok

File: recipes/pyorgmode/meta.yaml [hunk 9]
```diff
@@ -0,0 +1,37 @@
...
+
+  run:
+    - python
+
+test:
+  imports:
+    - {{ name }}
+
+about:
+  home: https://github.com/bjonnh/PyOrgMode
```
Comment (jakirkham) [inline recipes/pyorgmode/meta.yaml]: So PyPI points to https://github.com/gonvaled/PyOrgMode . However, that is a fork of https://github.com/bjonnh/PyOrgMode , which is also pretty out-of-date. Interestingly both include the former as the home page.

Comment (jakirkham) [inline recipes/pyorgmode/meta.yaml]: @gonvaled and @bjonnh , could you please help us sort out which is the primary source for `PyOrgMode`?

Comment (jschueller) [inline recipes/pyorgmode/meta.yaml]: imho the pypi ilnk is wrong, we can see https://github.com/bjonnh/PyOrgMode has more commits and accepted PRs

**Comment by bjonnh** [comment]
Yes my version is the original/main one. I'm not using it much right now, but I try to keep the PR number low ;)

**Comment by jakirkham** [comment]
Thanks for the info. So I have tried to cleanup some of the metadata to make this clearer in PR ( https://github.com/bjonnh/PyOrgMode/pull/30 ). Please let me know if that is ok.

File: recipes/pyorgmode/meta.yaml [hunk 10]
```diff
@@ -0,0 +1,37 @@
...
+  license: GPLv3
+  # https://github.com/bjonnh/PyOrgMode/pull/29
+  # license_file: LICENSE
```
Comment (jakirkham) [inline recipes/pyorgmode/meta.yaml]: xref: https://github.com/bjonnh/PyOrgMode/pull/29

**Comment by jschueller** [comment]
@jakirkham good to go ?

**Comment by jschueller** [comment]
@scopatz this one is ready too.

**Comment by jschueller** [comment]
cc @adumasphi

**Pull request merged**